src/app/js/main.js

```diff
@@ -191,7 +191,25 @@ function submit_jira(slide) {
   slide.appendChild(element)
 }
 
-
+// call our server rest api 
+function submitReport(data) {
+  const button = document.getElementById('jiraBtn');
```

Comment:
also, possibly move this to the onclick event in the html for the button.

Reply:
yes, sounds good